### PR TITLE
Fix wrong path join of paths in hub

### PR DESCRIPTION
**Pull request by ofekby**

### CODE REVIEW

File: src/hub/hubUri.ts [hunk 1]
```diff
@@ -22,8 +23,8 @@ export default async function hubUri(
     );
   }
 
-  if (path) {
-    hubUrl.pathname += path.startsWith("/") ? path : `/${path}`;
+  if (hubPath) {
+    hubUrl.pathname = path.join(hubUrl.pathname, hubPath);
```
Comment (assapir) [inline src/hub/hubUri.ts:27]: That url path or fs path? If it's url, I think it's not the right lib to use

Comment (ofekby) [reply src/hub/hubUri.ts:27]: Path is a path is a path

Comment (assapir) [reply src/hub/hubUri.ts:27]: nope https://stackoverflow.com/a/16301947/5520074

Comment (assapir) [reply src/hub/hubUri.ts:27]: Because it won't work as expected in windows for example

Comment (ofekby) [reply src/hub/hubUri.ts:27]: This is not URL, it is a join between two paths, hubUrl.pathname and hubPath

Comment (assapir) [reply src/hub/hubUri.ts:27]: You are joining url paths. The path separator is \ in windows, so it will be miss behaving

Comment (ofekby) [reply src/hub/hubUri.ts]: Never mind the VSCODE extensions can't use this package anyway

**Review by assapir — APPROVED** [review]

**Pull request merged**